src/_static/css/custom.css

```diff
@@ -30,3 +30,19 @@ h3 {
     max-width: fit-content;
   }
 }
+
+/* Question dropdown stuff */
+/* see https://pydata-sphinx-theme.readthedocs.io/en/stable/user_guide/extending.html#combining-all-three-customizations */
+div.admonition.admonition-question {
+  /*border-color: hsl(0, 100%, 50%); /* custom colors etc */
+}
+div.admonition.admonition-question > .admonition-title:before {
+  /*background-color: hsl(0, 99%, 18%);*/
+}
```

Comment:
If we aren't actually going to customize styles, let's delete these empty rules.